node/src/actors/chain_manager/mining.rs

```diff
@@ -647,6 +662,13 @@ impl ChainManager {
                             // Mailbox error
                             .map_err(|e| log::error!("Couldn't run tally: {}", e))?;
 
+                        // RadError::Unknown should not be included in a Tally until second hard fork
```

Comment:
This is only enforced in mining, not in validations. So if a node disables this check and broadcasts a block with a RadError::Unknown tally and it will be accepted as valid before the hard fork date.

Reply:
A node that tries to validate a tally with a RadError::Unknown produced by a radon panic and it is not updated, it will panic so...